### PR TITLE
Pass through all parameters for multiquery through the HTTP Client

### PR DESCRIPTION
**Pull request by haardikk21**
# [Pass all parameters when using multiqueries through HTTP Client] - #[https://github.com/ceramicnetwork/js-ceramic/issues/1797]

## Description

https://github.com/ceramicnetwork/js-ceramic/issues/1797 described the case where loading a deterministic stream with provided genesis did not resolve while using multiqueries. However, it should have been. The issue arose due to the HTTP client not passing forth the `genesis` parameter to the ceramic daemon, which this PR fixes.

- [x] The work addresses the description and outcomes in the issue
- [x] I have added relevant tests for new or updated functionality
- [x] My code follows conventions, is well commented, and easy to understand
- [x] My code builds and tests pass without any errors or warnings
- [x] I have tagged the relevant reviewers
- [ ] I have updated the READMEs of affected packages
- [ ] I have made corresponding changes to the documentation
- [x] The changes have been communicated to interested parties

### CODE REVIEW

**Review by stbrody — COMMENTED** [review]
code change looks good, but we need to add some tests for this.  Can go in the 'multiqueries' section of ceramic-daemon.test.ts

**Review by PaulLeCam — APPROVED** [review]

**Review by stbrody — CHANGES_REQUESTED** [review]
needs tests

**Comment by haardikk21** [comment]
> needs tests
does https://github.com/ceramicnetwork/js-ceramic/pull/1802/commits/00d3fd51b9cad0c55013e67247206f95343d9da1 suffice?
cc @PaulLeCam

**Comment by PaulLeCam** [comment]
LGTM but up to @stbrody if anything else is needed

File: packages/cli/src/__tests__/ceramic-daemon.test.ts
Comment (stbrody) [inline packages/cli/src/__tests__/ceramic-daemon.test.ts:405]: nitpick: why not just put this in the original `metadata` object?

**Review by stbrody — APPROVED** [review]
LGTM, thanks!

File: packages/cli/src/__tests__/ceramic-daemon.test.ts
Comment (haardikk21) [reply packages/cli/src/__tests__/ceramic-daemon.test.ts:405]: Because `deterministic` is not part of the metadata when we receive the query response

```
expect(resCore[streamId.toString()].metadata).toEqual(metadata)
expect(resClient[streamId.toString()].metadata).toEqual(metadata)
```

Comment (stbrody) [reply packages/cli/src/__tests__/ceramic-daemon.test.ts:405]: ah yes, good call.  Sounds good

**Pull request merged**
